src/MSBuild/MSBuild.csproj

```diff
@@ -217,6 +217,7 @@
     <Reference Include="System.Core" />
     <Reference Include="System.Xml" />
     <Reference Include="System.Configuration" />
+    <PackageReference Include="LargeAddressAware" Version="$(LargeAddressAwareVersion)" PrivateAssets="All" />
```

Comment:
Shouldn't PrivateAssets be set to `build` ? https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets

As far as I understand from that documentation, only any of the `contentfiles;analyzers;build` values are consumed but do not flow to the parent projects

Reply:
Nevermind, read it wrong, `All` is good :)